.github/workflows/cd.yml

```diff
@@ -262,16 +262,20 @@ jobs:
       - uses: ./.github/actions/setup-node
       - name: Build release binaries
         run: |
+          # Build Windows Tauri GUI
+
           pnpm install
           pnpm build
 
           # Used for release artifact
           # This should match 'build-tauri' in _rust.yml
           cp "../target/release/Firezone.exe" "${{ env.BINARY_DEST_PATH }}-x64.exe"
           cp "../target/release/bundle/msi/*.msi" "${{ env.BINARY_DEST_PATH }}-x64.msi"
+          cp "../target/release/firezone_windows_client.pdb" "${{ env.BINARY_DEST_PATH }}-x64.pdb"
```

Comment:
I may be missing something, but it's probably a good idea to leave these out of the published release since those are user-facing.

```suggestion
```

Reply:
Okay. Is there somewhere private we can put them? It'll be hard to rebuild them after the fact if they aren't captured at the same time as the exe is built.

Reply:
oh okay nvm I see you only removed them from cd.yml, so they'll still be in the CI runs just not the releases page